revel.go

```diff
@@ -227,6 +227,17 @@ func getLogger(name string) *log.Logger {
 			output = os.DevNull
 		}
 
+		logPath := filepath.Dir(output)
+		if _, err := os.Stat(logPath); err != nil {
+			if os.IsNotExist(err) {
+				if err := os.MkdirAll(logPath, 0777); err != nil {
```

Comment:
I agree that `0755` is better than `0777`. But even better option is to inherit CHMOD from a parent directory.